src/main/java/io/vertx/core/spi/cluster/RegistrationInfo.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2011-2019 Contributors to the Eclipse Foundation
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ */
+
+package io.vertx.core.spi.cluster;
+
+import java.util.Objects;
+
+/**
+ * Registration data stored by the cluster manager.
+ *
+ * @author Thomas Segismont
+ */
+public final class RegistrationInfo {
+
+  private final String nodeId;
+  private final long seq;
+  private final boolean localOnly;
+
+  public RegistrationInfo(String nodeId, long seq, boolean localOnly) {
+    Objects.requireNonNull(nodeId, "nodeId is null");
+    this.nodeId = nodeId;
+    this.seq = seq;
+    this.localOnly = localOnly;
+  }
+
+  public String getNodeId() {
```

Comment:
let's use same convention and name it `nodeId()`

Reply:
Done